src/test/java/org/terasology/logic/health/BlockDamageRendererTest.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.logic.health;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BlockDamageRendererTest {
+
+    BlockDamageRenderer blockDamageRenderer = new BlockDamageRenderer();
+
+    static HealthComponent healthComponent(int current, int max) {
+        HealthComponent health = new HealthComponent();
+        health.maxHealth = max;
+        health.currentHealth = current;
+        return health;
+    }
+
+    static String display(HealthComponent component) {
+        return "Health(" + component.currentHealth + "/" + component.maxHealth + ")";
+    }
+
+    static Stream<Arguments> validHealthComponents() {
+        return Stream.of(
+                Arguments.of(0, healthComponent(120, 100)),
+                Arguments.of(0, healthComponent(100, 100)),
+                Arguments.of(0, healthComponent(96, 100)),
+                Arguments.of(1, healthComponent(95, 100)),
+                Arguments.of(1, healthComponent(90, 100)),
+                Arguments.of(9, healthComponent(6, 100)),
+                Arguments.of(10, healthComponent(5, 100)),
+                Arguments.of(10, healthComponent(0, 100))
+        );
+    }
+
+    static Stream<Arguments> invalidHealthComponents() {
+        return Stream.of(
+                Arguments.of(healthComponent(-20, 100)),
+                Arguments.of(healthComponent(-20, -100)),
+                Arguments.of(healthComponent(20, -100))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("validHealthComponents")
+    void getDamageEffectsNumber_validHealthComponents(int expected, HealthComponent health) {
```

Comment:
This follows the naming scheme as defined in [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names).

Reply:
is there a way to add this to the linting to it doesn't get marked.

Reply:
probably, but in the meantime I found a way to make better use of `@DisplayName` and would not use this format anymore... will change it and merge then :nerd_face: